### ISSUE TITLE
Events (AE sets) Permission

### ISSUE THREAD

**Issue by pricelessdev** [GC-spigot/AdvancedEnchantments#2905]
### Describe the bug

AE Version 8.11.6, Minecraft Version 1.18.2.

Having trouble using asets 

`events:
  EFFECT_STATIC:
    chance: 100
    effects:
      - 'POTION:SPEED:5'
      - 'POTION:HEALTH_BOOST:1'
      - 'PERMISSION:wasteland.part.two'
  SPRINT:
    chance: 100
    effects:
      - 'PARTICLE:HEART:15:15'
  ATTACK:
    chance: 100
    effects:
      - 'INCREASE_DAMAGE:1'
  ATTACK_MOB:
    chance: 100
    effects:
      - 'INCREASE_DAMAGE:2'`
The permission that is made (using luckperms) adds context world: world, aka the world name you are in when equipping the last piece. If you leave the world and unequip elsewhere the permission does not get deleted from the old world and having the context of world and/or server is pointless in most cases. 

In summary, the permission needed can't contain context like the world and/or server names as the permission won't be deleted if you unequip the same gear when changing server and/or the world. 

### How to reproduce

Use the code above in when full set equipped gives permission, and then equip, change world and unequip to see if the permission is deleted.

### "/ae plinfo" link

https://paste.md-5.net/ruwamuxeco

### Server Log

https://mcpaste.io/603cc5022e56b3a7
